index.js

```diff
@@ -116,11 +126,14 @@ class Bookend extends BookendInterface {
      * @param  {PipelineModel}  o.pipeline  Pipeline model for the build
      * @param  {JobModel}       o.job       Job model for the build
      * @param  {Object}         o.build     Build configuration for the build (before creation)
+     * @param  {String}         clusterName Cluster name
```

Comment:
brackets for optional param
```suggestion
     * @param  {String}         [clusterName] Cluster name
```